pkg/database/rql/Handler.go

```diff
@@ -0,0 +1,149 @@
+package rql
+
+import (
+	"encoding/json"
+	"github.com/quan-to/chevron/internal/tools"
+	"github.com/quan-to/slog"
+	r "gopkg.in/rethinkdb/rethinkdb-go.v6"
+	"strings"
```

Comment:
Same comment for [imports](https://github.com/golang/go/wiki/CodeReviewComments#imports).